lib/cli.js

```diff
@@ -120,3 +129,13 @@ npmCheck(options)
         }
         process.exit(1);
     });
+
+function detectPreferredInstaller(cwd) {
+    return detectPreferredPM(cwd)
+        .then(preferredPM => {
+            if (preferredPM && preferredPM.name === 'pnpm') {
```

Comment:
I think it would be nice if this would accept e.g. `yarn` as well, and not specifically check for one manager...